public/Aframe/samples/loading-scene-1/scene-1.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>Loading Room 1</title>
+        <meta name="description" content="Loading Room 1" />
+        <script src="https://aframe.io/releases/1.1.0/aframe.min.js"></script>
+        <script src="https://cdn.jsdelivr.net/gh/donmccurdy/aframe-extras@v6.1.0/dist/aframe-extras.min.js"></script>
+        <script src="//cdn.rawgit.com/donmccurdy/aframe-physics-system/v4.0.1/dist/aframe-physics-system.min.js"></script>
```

Comment:
This seems to be causing an _infinite_ number of error/warning messages in the console (I had it running for 5 min and had 20k+ messages before chrome crashed)